network_analysis/configure-subnetwork-trace/README.md

```diff
@@ -0,0 +1,69 @@
+# Configure subnetwork trace
+
+Get a server-defined trace configuration for a given tier and modify its traversability scope, add new condition barriers and control what is included in the subnetwork trace result.
+
+![Configure Subnetwork Trace Sample Screenshot](ConfigureSubnetworkTrace.png)
+
+## Use case
+
+While some traces are built from an ad-hoc group of parameters, many are based on a variation of the trace configuration taken from the subnetwork definition. For example, an electrical trace will be based on the trace configuration of the subnetwork, but may add additional clauses to constrain the trace along a single phase. Similarly, a trace in a gas or electric design application may include features with a status of "In Design" that are normally excluded from trace results.
+
+## How to use the sample
```

Comment:
I think there should be a note in here that you have to use the "Add" button in order to add it and then run using the trace button.